Product Manager Description & Project [JOB EXPLORER]

LuckyHariharan opened the pull request:
# 🚀 Add Product Manager Job Explorer Feature

## 📘 In this PR I:
Added the **Product Manager Job Explorer**, including  manadtory skills, additional skills, and a project page with instructions and ideas to get starteed on their first product manager side project.

---

### **Resources**

[Roadmap.sh](https://roadmap.sh/product-manager) 

 [Curtis' Job Explorer Scoping Document](https://docs.google.com/document/d/1Lpb1-Uq4TgKJ9Z6lTLlu7Gt1QOYscgDHrw4m4okEks4/edit?tab=t.0)

### **Mandatory Skills**
- Idea generation
- Market and user research
- Product strategy and planning
- Product design collaboration
- Development, launch, and metrics analysis
- Stakeholder management

---

### **Additional Skills**
- Agile methodologies
- Data analysis tools
- Technical literacy
- Pricing strategies
- Leadership and presentation skills

---

![image](https://github.com/user-attachments/assets/f1219552-2dbf-44df-b74d-b09aa00439d0)
![image](https://github.com/user-attachments/assets/9b6c2b72-5408-43fc-ae89-47c6109fa830)

LuckyHariharan commented on pages/resources/jobExplorer/productManager/index.tsx:
```diff
@@ -1,7 +1,6 @@
-
+import LandingNavbar from "../../../../components/landingPage/LandingNavbar";
+import ProductManagerComponent from "../../../../components/resources/jobExplorer/productManager/ProductManagerComponent";
 export default function ProductManagerPage() {
```
@Curtoista  I used `index.tsx` files inside the folder name `productManager`

We should align on how you architect this page, because I think I'm doing it differently to you.

LuckyHariharan replied:
reverted, as per our OH convo yesterday

Curtoista commented on pages/resources/jobExplorer/productManagerProject.tsx:
```diff
@@ -0,0 +1,10 @@
+    <ProductManagerProject/>
+  )
+}
+
```
Need to add the formatting in line with other project page layouts

Curtoista commented on pages/resources/jobExplorer/productManagerProject.tsx:
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import ProductManagerProject from '../../../components/resources/jobExplorer/productManager/ProductManagerProject'
+
+const ProductManagerPage = () => {
```
Edit this to product manager project page

Curtoista commented on pages/resources/jobExplorer/productManager.tsx:
```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import LandingNavbar from "../../../components/landingPage/LandingNavbar";
 import ProductManagerComponent from "../../../components/resources/jobExplorer/productManager/ProductManagerComponent";
-
```
Do we need this change?

Curtoista reviewed (commented):
This looks solid, a few minor changes but everything seems good. Just stay consistent with other modules for different career paths

Curtoista commented:
Dont forget to remove product manager from the disabled jobs array

LuckyHariharan commented:
> Dont forget to remove product manager from the disabled jobs array

Thanks for thorough review Curtis! Addressed all of your comments

- removed unnecessary change to `productManager.tsx`
- corrected `ProductManageferProjectPage `name
- added get layout function to project page
- removed product manager from `disabledJobs`

@vithushan19 ready for your final review

vithushan19 commented on components/resources/jobExplorer/jobList/JobListComponent.tsx:
```diff
@@ -2,7 +2,6 @@ import Image from "next/legacy/image";
 import React from "react";
 
 const disabledJobs = [
-  "Product Manager",
```
Can we just delete the entire array now? If it's not used

Curtoista replied:
Yeah, were gonna delete this. This whole component is gonna be refactored to remove the isDisabled function as all links are active now

vithushan19 reviewed (approved):
Approving to unblock. But I don't think it's so hard to use the reusable component for the project page. Up to you, since you already implemented it this way and it's working fine. 

You can refactor later if you'd like.
